README.md

```diff
@@ -114,6 +114,26 @@ Use it at your own risk.
 
 Please see the [LICENSE](https://github.com/pfnet-research/chainer-chemistry/blob/master/LICENSE) file for details.
 
+## Links
+
+Links for Chainer Chemistry:
+
+ - Document: [https://chainer-chemistry.readthedocs.io](https://chainer-chemistry.readthedocs.io)
+ - Blog: [Release Chainer Chemistry: A library for Deep Learning in Biology and Chemistry](https://preferredresearch.jp/2017/12/18/chainer-chemistry-beta-release/)
+
+Links for other Chainer project:
+
+ - Chainer: A flexible framework of neural networks for deep learning - [chainer/chainer](https://github.com/chainer/chainer)
+ - ChainerRL: Deep reinforcement learning library built on top of Chainer - [chainer/chainerrl](https://github.com/chainer/chainerrl)
+ - ChainerCV: A Library for Deep Learning in Computer Vision - [chainer/chainercv](https://github.com/chainer/chainercv)
+ - ChainerMN: Scalable distributed deep learning with Chainer - [chainer/chainermn](https://github.com/chainer/chainermn)
+ - ChainerUI: User Interface for Chainer - [chainer/chainerui](https://github.com/chainer/chainerui)
+ - PaintsChainer: Line drawing colorization using chainer - [pfnet/PaintsChainer](https://github.com/pfnet/PaintsChainer)
```

Comment:
Which do you think it is better to use the demo URL (https://paintschainer.preferred.tech/) or github repository (https://github.com/pfnet/PaintsChainer) as a destination URL?

Reply:
Maybe we can put both.